pouchdb-core/pouchdb-core.d.ts

```diff
@@ -64,12 +75,62 @@ declare namespace PouchDB {
                 ids: RevisionId[];
                 start: number;
             }
+
+            /** Attachments where index is attachmentId */
+            _attachments?: Attachments;
+        }
+
+        interface AttachmentResponse {
+            content_type: string;
+
+            /** Hash ? */
+            digest: string;
+
+            /** Only present if `attachments` was `false`. */
+            stub?: boolean;
+
+            /** Only present if `attachments` was `false`. */
+            length?: number;
+
+            /** 
+             * Only present if `attachments` was `true`.
+             * {string} if `binary` was `false`
+             * {Blob|Buffer} if `binary` was `true`
+             */
+            data?: string | Blob | Buffer;
         }
+
+        interface Attachments {
+            [attachmentId: string]: AttachmentResponse;
+        }
+
         type NewDocument<Content extends Encodable> = Content;
         type Document<Content extends Encodable> = Content & IdMeta;
         type ExistingDocument<Content extends Encodable> =
                 Document<Content> & RevisionIdMeta;
 
+        /** Pass existing doc, or just object with `_id` and `_rev` */
```

Comment:
This will show up any time this type is used, so I would suggest removing "Pass", and e.g. saying "Object with a document's `_id` and `_rev` fields".